### ISSUE TITLE
Offline_Access permission not included in Token even though specified in scopes and API permissions of Azure app

### ISSUE THREAD

**Issue by xmarshallx** [O365/python-o365#611]
I'm attempting to use the o365 module to connect to an O365 Exchange mailbox. I can  complete the oauth process to get a token and make the connection to the mailbox successfully however the token issued does not include the offline_access permission and as a result the token only lasts 1 hour. 

My Azure application has the  offline_access API permission assigned. Additionally I have specificed the offline_access scope in my python code. When I copy paste the auth URL and complete the user consent portion I can see the offline access perm in the list of permissions on the pop up consent prompt -  but the token returned simply does not add the offline_access permission.

A very simplified sample code to reproduce the issue. 

**import O365
from O365 import Account

credentials = ('-ClientIDHere-', '-clientSecretHere-')
account = Account(credentials, tenant_id='-TenantIDHere-')
account.authenticate(scopes=['User.Read', 'Mail.Send', 'Mail.ReadWrite', 'Mail.Read', 'OFFLINE_ACCESS'])**

I'm then prompted to copy and paste the Auth URL into a browser. I complete this step successfully and copy back the URL which gives me a successful **"Authentication Flow Completed. Oauth Access Token Stored. You can now use the API."**

When I view the token however via:    **account.connection.token_backend.token**  The token does not contain offline access. Instead it lists the following scopes:

**{'token_type': 'Bearer', 'scope': ['Mail.Read', 'User.Read', 'profile', 'openid', 'email', 'Mail.ReadWrite', 'Mail.Send'], 'expires_in': 3599, 'ext_expires_in': 3599,**

After an hour the token stops working.

Do you have any idea why the Offline_Access scope is not being included in the token ? Is there another setting in the code or the Azure app that needs to be set ?

**Comment by xmarshallx** [comment]
Sorry I think this issue may be erroneous. I noticed I had an unseen error in the path for the tokenbackend. A space character was included in the path when loading the token file from disk. When I corrected this the token is now working. 

Just wanted to confirm though - is it normal that the scopes listed out in the top of the token file will not show Offline_Access? As my token shows all other permissions listed - but not this one.
